pom.xml

```diff
@@ -411,7 +406,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-dependency-plugin</artifactId>
-                    <version>3.3.0</version>
```

Comment:
My typical modus operandi is to update all the plugins and library versions as part of the process of doing a release. (It's even documented that way in the [ESAPI Release Steps](https://github.com/ESAPI/esapi-java-legacy/blob/develop/documentation/ESAPI-release-steps.pdf).) That saves work for us because otherwise we are updating the pom.xml multiple times instead of just once.  And in the past, Dependabot, Snyk, etc. have created PRs that caused the tests to fail. [You'd think they would try to at least run the tests before submitting the PR, but apparently do not. Sigh.]

Anyway, I am okay with _you_  doing this, but I really don't want to set a precedent, because it just gets to be a distraction from the main point of the code review. So, for this particular case, I don't want you to undo things (except where otherwise noted), but I'd recommend skipping this in the future.